experiments/run_landcoverai_experiments.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Runs the train script with a grid of hyperparameters."""
+import itertools
+import os
+import subprocess
+from multiprocessing import Process, Queue
+
+# list of GPU IDs that we want to use, one job will be started for every ID in the list
+GPUS = [0, 1, 2, 3, 4, 5, 6, 7]
+TEST_MODE = False  # if False then print out the commands to be run, if True then run
```

Comment:
A better name would be `DRY_RUN` since that's what other programs call this option.